File: php/fibonacci.php
<?php

function fibonacci() {
	$a = $b = 1;
	$temp = 0;
	while(true) {
		$temp = $a; $a = $b; $b = $temp + $b;
		if(factores($a) >= 1000) {
			print $a;
			return false;
		}
	}
}

function factores($num) {
	$divisores = [];
	for($i = 1; $i <= sqrt($num); ++$i) {
		if($num % $i == 0) {
			$divisores[] = $i;
			if($i != $num/$i) {
				$divisores[] = $num/$i;
			}
		}
	}
	return count($divisores);
}

fibonacci();

File: swift/fibonacci.swift
import Darwin

func fibonacci() {
    var a: Double = 1; var b: Double = 1; var temp: Double;
    var finish :Bool = false
    while(!finish) {
        temp = a; a = b; b += temp;
        if(factores(a) >= 1000) {
            print(Int(a))
            finish = true
        }
    }
}

func factores(num: Double) -> Int {
    var divisores = [Double]()
    var n: Double = 1
    while(n <= sqrt(num)) {
        if(num % n == 0) {
            divisores.append(n)
            if(n != num/n) {
                divisores.append(num/n)
            }
        }
        n++
    }
    return divisores.count
}

fibonacci()